pom.xml

```diff
@@ -85,7 +85,7 @@
     <!-- properties for script build step that creates the config files for the artifacts -->
     <widgets.dir>widgets</widgets.dir>
     <docs.dir>docs</docs.dir>
-    <etl.versionRange>[4.0.0,10.0.0-SNAPSHOT)</etl.versionRange>
+    <app.parents>system:cdap-etl-batch[4.1.0,4.2.0-SNAPSHOT),system:cdap-etl-realtime[4.1.0,4.2.0-SNAPSHOT),system:cdap-data-pipeline[4.1.0,4.2.0-SNAPSHOT),system:cdap-data-streams[4.1.0,4.2.0-SNAPSHOT)</app.parents>
```

Comment:
Unless somethimg breaks (I know there was some plugin function that required this) without this, lets not add it